README.md

```diff
@@ -262,16 +262,22 @@ that of `nb-clean` >=2.0.0.
+| Description                                                  | `nb-clean` 1.6.0                                                    | `nb-clean` 2.4.0                                                         |
+| ------------------------------------------------------------ | ------------------------------------------------------------------- | ------------------------------------------------------------------------ |
+| Clean notebook                                               | `nb-clean clean -i/--input notebook.ipynb \| sponge notebook.ipynb` | `nb-clean clean notebook.ipynb`                                          |
+| Clean notebook (remove empty cells)                          | `nb-clean clean -i/--input notebook.ipynb -e/--remove-empty`        | `nb-clean clean notebook.ipynb -e/--remove-empty-cells`                  |
+| Clean notebook (preserve cell metadata)                      | `nb-clean clean -i/--input notebook.ipynb -m/--preserve-metadata`   | `nb-clean clean notebook.ipynb -m/--preserve-cell-metadata`              |
+| Clean notebook (preserve `tags` and `special` cell metadata) |                                                                     | `nb-clean clean notebook.ipynb -m/--preserve-cell-metadata tags special` |
+| Clean notebook (preserve cell outputs)                       |                                                                     | `nb-clean clean notebook.ipynb -o/--preserve-cell-outputs`               |
+| Clean notebook (remove notebook metadata)                    |                                                                     | `nb-clean clean notebook.ipynb -M/--remove-notebook-metadata`            |
+| Check notebook                                               | `nb-clean check -i/--input notebook.ipynb`                          | `nb-clean check notebook.ipynb`                                          |
+| Check notebook (ignore non-empty cells)                      | `nb-clean check -i/--input notebook.ipynb -e/--remove-empty`        | `nb-clean check notebook.ipynb -e/--remove-empty-cells`                  |
+| Check notebook (ignore cell metadata)                        | `nb-clean check -i/--input notebook.ipynb -m/--preserve-metadata`   | `nb-clean check notebook.ipynb -m/--preserve-cell-metadata`              |
+| Check notebook (ignore `tags` and `special` cell metadata)   |                                                                     | `nb-clean check notebook.ipynb -m/--preserve-cell-metadata tags special` |
+| Check notebook (ignore cell outputs)                         |                                                                     | `nb-clean check notebook.ipynb -o/--preserve-cell-outputs`               |
+| Check notebook (ignore notebook metadata)                    |                                                                     | `nb-clean check notebook.ipynb -M/--remove-notebook-metadata`            |
```

Comment:
This doesn't look to be fully rebased: there aren't short flags such as `-i` in the nb-clean 1.6.0 column on `main`.